install/install.qmd

```diff
@@ -141,56 +142,96 @@ TODO 11
 
 <script src="installer.js"></script>
 
-## In-Depth Instructions
 
-### Platform
 
-#### Linux
+## Installers
 
-- which C++
 
-#### MacOS
+#### Native installers:
```

Comment:
I don't think this re-organization is helpful to users; it reads like how we, as Stan developers, think of things, not how a user would.

e.g., there is no reason that "install.packages" and "pip" should be mentioned in the same section -- users of one will almost assuredly not care about the other, or at least not enough that it will be helpful

Reply:
good point.  simplified.